Makefile

```diff
@@ -41,49 +45,42 @@ ifneq ($(HOSTARCH),$(ZARCH))
 CROSS = 1
 $(warning "WARNING: We are assembling a $(ZARCH) image on $(HOSTARCH). Things may break.")
 endif
-# canonicalized names for architecture
-ifeq ($(ZARCH),aarch64)
-        ZARCH=arm64
-endif
-ifeq ($(ZARCH),x86_64)
-        ZARCH=amd64
-endif
 
-QEMU_SYSTEM_arm64:=qemu-system-aarch64
-QEMU_SYSTEM_amd64:=qemu-system-x86_64
-QEMU_SYSTEM=$(QEMU_SYSTEM_$(ZARCH))
+DOCKER_ARCH_TAG=$(ZARCH)
+
+# EVE rootfs image manifest
+ROOTFS_YML_xen_amd64=images/rootfs-xen.yml
+ROOTFS_YML_xen_arm64=images/rootfs-xen.yml
+ROOTFS_YML_acrn_amd64=images/rootfs-acrn.yml
+ROOTFS_YML_acrn_arm64=/dev/null
```

Comment:
There is a non-zero risk that this will appear in a clean rule, and then someone will run sudo make clean and be in a lot of pain due to loosing /dev/null.